Fw/Types/StringUtils.cpp

```diff
@@ -29,3 +29,27 @@ U32 Fw::StringUtils::string_length(const CHAR* source, U32 max_len) {
     }
     return length;
 }
+
+I32 Fw::StringUtils::substring_find(const CHAR* source_string, U32 source_size, const CHAR* sub_string, U32 sub_size) {
```

Comment:
We should use` FwSignedSizeType` instead of I32 and U32 here.

Reply:
Fixed